paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -285,21 +285,29 @@ public class PaparazziSdk @JvmOverloads constructor(
       viewGroup.addView(modifiedView)
       for (frame in 0 until frameCount) {
         val nowNanos = (startNanos + (frame * 1_000_000_000.0 / fps)).toLong()
-        withTime(nowNanos) {
-          val result = renderSession.render(true)
-          if (result.status == ERROR_UNKNOWN) {
-            throw result.exception
+        var result = withTime(nowNanos) {
```

Comment:
I like isolating just the render call in the `withTime {}` block, it makes it feel less cluttered